core/domain/stats_services.py

```diff
@@ -122,6 +169,14 @@ def record_answers(
         submitted_answer_list):
     """Optimally record a group of answers using an already loaded exploration..
     The submitted_answer_list is a list of SubmittedAnswer domain objects.
+
+    Args:
+        exploration_id: str. The exploration ID.
+        exploration_version: int. The version of the exploration.
```

Comment:
Test files are fine, yep.

I have a feeling you're not synced to the latest version of develop. Try merging develop into your branch (see step 5 [here](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change)) and then run `grep get_state_answers core -r`. But remember, you can also read the other callsites and use general reasoning to figure out what is going on.